player.go

```diff
@@ -19,7 +19,8 @@ import (
 	"time"
 )
 
-// A Player is a sound player.
+// Player is a PCM (pulse-code modulation) audio player. It implements io.Writer, use Write method
```

Comment:
As we discussed, we learned that articles are optional:

https://github.com/golang/lint/issues/11